### PR TITLE
Debugged File Sending

### PR DESCRIPTION
**Pull request by nnk03**
Changed Chunk Size to 1 MB, changed default folder path to `Temp` and now resizing files to adjust for the final chunk

Yet to test it in machines

### CODE REVIEW

File: Network/Sockets/SocketListener.cs [hunk 1]
```diff
@@ -181,18 +181,25 @@ private string ProcessReceivedString(string receivedString)
...
-                    packetString);
-                _receivingQueue.Enqueue(packet);
-
-                // remove the first packet from the received string
-                receivedString = receivedString[(packetEnd + 6)..];
-
-                Trace.WriteLine("[Networking] SocketListener:" +
-                    " Received data from module: "
-                    + packet.moduleOfPacket);
+                try
```
Comment (dshreddy) [inline Network/Sockets/SocketListener.cs:184]: Do git restore on this file

File: FileCloner/Models/Constants.cs [hunk 2]
```diff
@@ -31,7 +31,7 @@ public class Constants
 
         // File & Folder Paths
         public static readonly string configFilePath = Path.GetFullPath(Path.Combine("..", "..", "..", "Assets", "Files", "config.txt"));
-        public static readonly string defaultFolderPath = Path.GetFullPath(Path.Combine("..", "..", "..", "Assets"));
+        public static readonly string defaultFolderPath = Path.GetFullPath(Path.Combine("..", "..", "..", "Temp"));
```
Comment (dshreddy) [inline FileCloner/Models/Constants.cs:34]: I think the "Temp" folder is not getting pushed probably because it's empty, add some dummy file & push

Comment (nnk03) [reply FileCloner/Models/Constants.cs:34]: done

File: Network/Sockets/SocketListener.cs (hunk 1 above)
Comment (nnk03) [reply Network/Sockets/SocketListener.cs]: I have manually removed the try catch block, it's better that you also check before merging

**Comment by nnk03** [comment]
Tested on @nnk03 and @sarath-a-12 machine
Ready to merge, (Be cautious about Networking module change, I have manually removed the try catch block, which we introduced for the purposes of debugging)

**Pull request merged**